common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/AuthorizationFragment.java

```diff
@@ -54,11 +54,6 @@ public abstract class AuthorizationFragment extends Fragment {
 
     private static final String TAG = AuthorizationFragment.class.getSimpleName();
 
-    /**
-     * The bundle containing values for initializing this fragment.
-     */
-    private Bundle mInstanceState;
```

Comment:
I think we still can use this, but moving to getArguments() is probably less confusing IMO.